bitnami/rabbitmq/templates/secrets.yaml

```diff
@@ -28,7 +28,7 @@ apiVersion: v1
 kind: Secret
 metadata:
   name: {{ $key }}
-  namespace: {{ $.Release.Namespace }}
+  namespace: {{ $.Release.Namespace | quote }}
```

Comment:
In theory, quotes shouldn't be necessary. In any case, they make no harm.

Reply:
it make my CI test to fail 
```
Error: YAML parse error on myproject/charts/rabbitmq/templates/secrets.yaml: error converting YAML to JSON: yaml: invalid map key: map[interface {}]interface {}{"demoEnvironment.namespace":interface {}(nil)}
```